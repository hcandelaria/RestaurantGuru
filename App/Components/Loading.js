import React, { Component } from 'react'
import { ActivityIndicator, View, Text} from 'react-native'

const styles = {
  container: {
    flex: 1,
    justifyContent: 'center',
    flexDirection: 'row',
    justifyContent: 'space-around',
    padding: 10
  }
}
export default class Loading extends Component {
  render () {
    return (
      <View style={styles.container}>
        <ActivityIndicator size='large' color='#0000ff' />
      </View>
    )
  }
}
