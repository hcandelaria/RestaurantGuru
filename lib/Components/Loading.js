var _interopRequireDefault=require("@babel/runtime/helpers/interopRequireDefault");var _interopRequireWildcard=require("@babel/runtime/helpers/interopRequireWildcard");Object.defineProperty(exports,"__esModule",{value:true});exports["default"]=void 0;var _classCallCheck2=_interopRequireDefault(require("@babel/runtime/helpers/classCallCheck"));var _createClass2=_interopRequireDefault(require("@babel/runtime/helpers/createClass"));var _possibleConstructorReturn2=_interopRequireDefault(require("@babel/runtime/helpers/possibleConstructorReturn"));var _getPrototypeOf2=_interopRequireDefault(require("@babel/runtime/helpers/getPrototypeOf"));var _inherits2=_interopRequireDefault(require("@babel/runtime/helpers/inherits"));var _defineProperty2=_interopRequireDefault(require("@babel/runtime/helpers/defineProperty"));var _react=_interopRequireWildcard(require("react"));var _reactNative=require("react-native");var _container,_jsxFileName="C:\\Users\\Smilodon\\Interviews\\bin2rid\\RestaurantGuru\\App\\Components\\Loading.js";var styles={container:(_container={flex:1,justifyContent:'center',flexDirection:'row'},(0,_defineProperty2["default"])(_container,"justifyContent",'space-around'),(0,_defineProperty2["default"])(_container,"padding",10),_container)};var Loading=function(_Component){(0,_inherits2["default"])(Loading,_Component);function Loading(){(0,_classCallCheck2["default"])(this,Loading);return(0,_possibleConstructorReturn2["default"])(this,(0,_getPrototypeOf2["default"])(Loading).apply(this,arguments));}(0,_createClass2["default"])(Loading,[{key:"render",value:function render(){return _react["default"].createElement(_reactNative.View,{style:styles.container,__source:{fileName:_jsxFileName,lineNumber:17}},_react["default"].createElement(_reactNative.ActivityIndicator,{size:"large",color:"#0000ff",__source:{fileName:_jsxFileName,lineNumber:18}}));}}]);return Loading;}(_react.Component);exports["default"]=Loading;