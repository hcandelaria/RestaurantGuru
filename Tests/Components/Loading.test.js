import 'react-native'
import React from 'react'
import Loading from '../../App/Components/Loading'
import renderer from 'react-test-renderer'

test('Loading component renders correctly', () => {
  const tree = renderer.create(<Loading/>).toJSON()
  expect(tree).toMatchSnapshot()
})
