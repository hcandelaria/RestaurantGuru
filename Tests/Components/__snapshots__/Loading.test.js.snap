// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "space-around",
      "padding": 10,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#0000ff"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
